packages/adapter-ccxt/src/validation/StepConfigValidator.ts

```diff
@@ -1 +1,119 @@
-export class StepConfigValidator {}
+import { injectable } from "tsyringe";
```

Comment:
See comment above re `zod` this could probably result in less code. Also made a gist to see how custom async zod validations could work with tsyringe: https://gist.github.com/bowd/bb150aea5403d8c183e6ad921d942633